### DISCUSSION TITLE
Allow opening files in text editors/IDE not in the shortlist

### DISCUSSION THREAD

**Discussion by pascalchevrel** [spatie/ray#59] [Ideas]
Currently there is a limited list of text editors that can by used with Ray, some of these are mac specific (TextMate, macVim). I would like to be able to provide a text editor executable path that is not in this shortlist as there are many options out there that are not listed (Eclipse, Vim, Geany, Komodo…)

**Comment by AdrianMrn** [comment]
See my reply here: https://github.com/spatie/ray/discussions/68#discussioncomment-274086

Feel free to suggest some editors that are not in the list yet! I'll add the 4 you mentioned here already!
